Free_Games.py

```diff
@@ -152,7 +157,7 @@ def start_firefox_browser(user_agent):
     profile = webdriver.FirefoxProfile()
     profile.set_preference("general.useragent.override", user_agent) # We will use the user-agent to trick the website into thinking we are a real person. This usually subverts most basic security
 
-    browser = webdriver.Firefox(profile) # Setup the browser object to use our modified profile
+    browser = webdriver.Firefox(profile, service_log_path="/config/geckodriver.log") # Setup the browser object to use our modified profile

```

Comment:
i need to add a is docker piece of code that toggle stuff like this so it works inside and outside of docker via the .env file